nova/compute/api.py

```diff
@@ -4187,6 +4187,26 @@ def resize(self, context, instance, flavor_id=None, clean_shutdown=True,
         host_name can be set in the cold migration case only.
         """
 
+        # get all volumes attached to the instance and check if they are in a
+        # valid state for resize
+        bdms = objects.BlockDeviceMappingList.get_by_instance_uuid(
+                context, instance.uuid)
+        for bdm in bdms:
+            if not bdm.volume_id:
+                continue
+            volume = self.volume_api.get(context, bdm.volume_id)
+            migration_status_valid = volume['migration_status'] != 'migrating'
+            status_and_attach_status_valid =  \
+                ((volume['status'] == "in-use" and
```

Comment:
If you move the opening `(` into the line before, you don't need the `\` anymore. IIRC, upstream doesn't like using `\` and we try to mimic upstream's coding style.